python/tk_multi_workfiles/entity_tree/entity_tree_form.py

```diff
@@ -627,9 +627,8 @@ def _on_data_refreshed(self, modifications_made):
         # try to select the current entity from the new items in the model:
         prev_selected_item = self._reset_selection()
         self._update_selection(prev_selected_item, True)
-        # Hide the overlay widget, if it was visible
-        if not self._refresh_overlay_widget.isHidden():
-            self._refresh_overlay_widget.hide()
+        # Hide the overlay widget
+        self._refresh_overlay_widget.hide()
```

Comment:
Removed the `isHidden` test as it didn't seem to work, so we now systematically hide the overlay widget, even if it's already hidden. Any worry, let me know!

Reply:
seems fine. i like explict. :)